egs/ljspeech/TTS/vits/vits.py

```diff
@@ -38,6 +39,36 @@
     "hifigan_multi_scale_multi_period_discriminator": HiFiGANMultiScaleMultiPeriodDiscriminator,  # NOQA
 }
 
+LOW_CONFIG = {
```

Comment:
The config values are from
https://github.com/rhasspy/piper/blob/master/src/python/piper_train/__main__.py#L68